src/main/java/com/ksm/wstbackoffice/controller/CountryController.java

```diff
@@ -44,4 +47,16 @@ public CountryController(CountryService countryService) {
         return ResponseEntity.status(HttpStatus.OK)
                 .body(country);
     }
+
+    @GetMapping("/filter-name")
```

Comment:
This request should look like this
GET
/countries?nameStartsWithFilters=uk, ame, blabla

Its response should look like
`
{
	"total": {
		"ba": 4,
		"tu": 2
	},
	"details": {
		"ba": ["Bahamas", "Bahrain", "Bangladesh", "Barbados"],
		"tu": ["Turkia", "Turkmenistan"]
	}	
}
`

You should test the result like this
1. response code = **200** or **400** or other
2. details.ba contains exactly in any order ["**Bahamas**", "**Bahrain**", "**Bangladesh**", "**Barbados**"]
3. details.to contains exactly in any order "tu": ["**Turkia**", "**Turkmenistan**"]
4. details total.ba equals **4**
5. details total.ba equals **2**
You should not hardcode any of the above-mentioned bold values, they should be passed through the argument parameters of the parametrized test.

You don't need multiple tests.
You should have one test or at max two parametrized tests: one for BAD_REQUEST and another for OK.

Reply:
done

Reply:
Uri is not correct. Plz check my comment before

Reply:
done